### DISCUSSION TITLE
Are windows actually monitored?

### DISCUSSION THREAD

**Discussion by Kevr0n228** [Hyundai-Kia-Connect/kia_uvo#352] [Q&A]
Looking at the bluelink app after the latest update rolled out to Canada and I now see a field for each of the 4 side windows on my Kona EV saying they're closed. I tried opening one and refreshing the data but they all still show as closed. Does this work for anyone else? If this is an actual working feature I'd love to be able to use it in the integration.

**Comment by cdnninja** [comment]
I didn’t think windows are a feature. Doors are though.  Your saying yours lists windows instead of doors?

**Reply by Kevr0n228** [reply]
![Screenshot_20220824-210531_Bluelink](https://user-images.githubusercontent.com/10468194/186550883-9dfa2c80-52c4-42ba-8814-d20a963656b0.jpg)
So this is what I get when I tap vehicle status, if I go to the "full list" option I can see a status for each window but when I tested it they never changed to open.

**Comment by cdnninja** [comment]
I should re read the first. I thought you meant our integration.  

If the window feature does work for you we can add to our integration. Maybe spend some time resting and let us know? 

My car doesn’t show windows as a feature.

**Comment by Kevr0n228** [comment]
If anyone else who has windows showing up in the bluelink app can confirm
whether or not they work, please comment! I'll try mine again tomorrow to
see if they change state as it seems bluelink is having a myriad of other
issues with the update.
